test/e2e/common_test.go

```diff
@@ -234,9 +234,18 @@ func PodmanTestCreateUtil(tempDir string, remote bool) *PodmanTestIntegration {
 	if os.Getenv("CONMON_BINARY") != "" {
 		conmonBinary = os.Getenv("CONMON_BINARY")
 	}
+
+	storageFs := STORAGE_FS
+	if rootless.IsRootless() {
+		storageFs = ROOTLESS_STORAGE_FS
+	}
+
 	storageOptions := STORAGE_OPTIONS
 	if os.Getenv("STORAGE_OPTIONS") != "" {
 		storageOptions = os.Getenv("STORAGE_OPTIONS")
+		if !strings.Contains(storageOptions, storageFs) {
```

Comment:
I would encourage you not to do it this way: it is not testing what the user expects to be tested, and would fail in all sorts of unexpected ways: for instance, if `storageFs` is "`vfs`" but `storageOptions` is "`--vfs-options xyz`"

Reply:
done